.github/workflows/webassembly.yml

```diff
@@ -50,4 +50,8 @@ jobs:
         with:
           node-version: "16"
       - run: npm ci
-      - run: npm run build
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+      - name: Run Playwright test
+        run: npm run e2e
+      - run: npm run build
```

Comment:
We should add a new line at the end of the file

Reply:
Thanks for remind, added a new line.